src/sign/index.ts

```diff
@@ -88,16 +88,17 @@ export default class Client {
         'Content-Type': 'application/json'
       },
       timeout: 20e3,
-      body: JSON.stringify(envelop)
+      body: envelop
     };
-    return new Promise((resolve, reject) => {
-      fetch(address, init)
-        .then((res) => {
-          if (res.ok) return resolve(res.json());
-          throw res;
-        })
-        .catch((e) => e.json().then((json) => reject(json)));
-    });
+
+    try {
+      return await fetch(address, init);
```

Comment:
you don't need `await` here

Reply:
I think there's a little difference when used inside a try/catch
https://jakearchibald.com/2017/await-vs-return-vs-return-await/#returning

Reply:
you are right, haven't thought about this